### PR TITLE
Formatting map spawners

### PR DESCRIPTION
**Pull request by ghost**
At first, this was the usual "Does not inherit New()" fix, but then I realized the code was awful so I decided to bring it up to gear
- Use lower_case argument and proc names and stop specifying .src scope everywhere. Let's be honest here, this isn't C++, we don't need to pretend like it is
- Misc formatting like putting spaces after commas because we aren't barbarians
- Moved code in New() to another proc so that New() can be inherited properly without causing problems. See, it wasn't that hard, now was it ?
- And of course, Defficiency used direct var edits. Thanks Duny

### CODE REVIEW

File: code/modules/maps/nests.dm [hunk 1]
```diff
@@ -5,58 +5,79 @@
...
+	var/pop_max = 30
 
-/obj/map/nest/New()
-	for(src.pop,src.pop,src.pop--)
-		new src.mobType(src.loc)
-	spawn() src.ticker()
-	return
+/obj/map/nest/perform_spawn()
+
+	for(pop, pop, pop--)
```
Comment (ComicIronic) [inline code/modules/maps/nests.dm]: This should be illegal.

No but really make this the regular kind of loop.

File: code/modules/maps/spawners/set_spawners.dm [hunk 2]
```diff
@@ -8,29 +8,28 @@
...
-			if(src.subChance)
-				new spawned(src.loc)
-				if(src.jiggle)
-					spawned.pixel_x = rand(-src.jiggle,src.jiggle)
-					spawned.pixel_y = rand(-src.jiggle,src.jiggle)
-	qdel(src)
-	return
-	
+	to_spawn = pick(to_spawn)
+	for(amount, amount, amount--)
```
Comment (ComicIronic) [inline code/modules/maps/spawners/set_spawners.dm]: ditto

**Comment by clusterfack** [comment]
It doesnt even compile you fuckin nerd

**Comment by ghost** [comment]
What happen

**Comment by clusterfack** [comment]
The lengths you people go through to accomplish nothing simply astounds me

**Pull request merged**